util/cmd/cmd.go

```diff
@@ -2,13 +2,16 @@
 package cmd
 
 import (
+	"flag"
 	"fmt"
 	"net/url"
 	"os"
+	"strconv"
 	"strings"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"k8s.io/klog"
```

Comment:
wrong klog

Reply:
this is why tests are failing again I think

Reply:
Yeah my bad on that - pushing the commit now